docs/mca/quickstart.rst

```diff
@@ -0,0 +1,77 @@
+.. _label-mca-quickstart:
+
+Quick start: Run-time tuning Open MPI
+=======================================
+
+Although this section skips many details, it offers examples that will
+probably work in many environments.
+
+.. caution:: Note that this section is a "Quick start" |mdash| it does
+   not attempt to be comprehensive or describe behavior in all
+   supported environments. The examples below therefore may not work
+   exactly as shown in your environment.
+
+   Please consult the other sections in this chapter for more details,
+   if necessary.
+
+Open MPI is highly configurable and can be configured during run-time
+using configuration files, command line parameters, and environment
+variables. The main configuration system is through the :ref:`Modular
+Component Architecture (MCA)<label-mca>`. It is highly advisable to
+read the linked section if any terms are unknown.
+
+MCA parameters
+--------------
+:ref:`MCA parameters<label-mca-terminology-parameters>` (sometimes
+called MCA *variables*) are the basic unit of run-time tuning for
+Open MPI.  They are simple "key = value" pairs that are used
+extensively throughout Open MPI.  The general rules of thumb that
+the developers use are:
+
+#. Instead of using a constant for an important value, make it an MCA
+   parameter.
+#. If a task can be implemented in multiple, user-discernible ways,
+   implement as many as possible, and use an an MCA parameter to
+   choose between them at run-time.
+
```

Comment:
I'd go ahead and delete this subsection from the quick start page.

Reply:
Sure